### ISSUE TITLE
Consent Management Module: submit dsa render info to external api

### ISSUE THREAD

**Issue by patmmccann** [prebid/Prebid.js#10958]
## Type of issue
Feature

## Description
enable pubrender of dsa information via the gpp

## Steps to reproduce
 prebid renderAd function would write info to a __gpp queue; and techlab would need to provide perhaps __gpp.provideDSAInfoForRenderedAd() in the gpp api and prebid would call it on render
This is blocked pending development of such an API

**Comment by janwinkler** [comment]
FYI: We (consentmanager) are going to already implement a solution for our customers, since there is a demand for it and clients do not want to wait for the final spec. We have summarized our implementation as a proposal for the IAB GPP working group here:
https://docs.google.com/document/d/15SnJB--56Ec5ag0DdeDYT8P6nU77DLh_gDa7l-mgRDQ/edit?usp=sharing

**Comment by patmmccann** [comment]
@janwinkler did you implement this? Did you propose it to a standards body? Did any other cmp implement?

**Comment by janwinkler** [comment]
yes we have implemented it as outlined in the working document, demo see here: https://www.consentmanager.net/en/dsa-transparency-demo/

**Comment by dgirardi** [comment]
Is it likely to become the standard? Do we want to pick this up @patmmccann ?
